enterprise/metrics/src/main/java/org/neo4j/metrics/output/GangliaOutput.java

```diff
@@ -56,42 +59,48 @@ public GangliaOutput( Config config, MetricRegistry registry, Log logger, String
     @Override
     public void init() throws IOException
     {
-        if ( config.get( gangliaEnabled ) )
-        {
-            // Setup Ganglia reporting
-            hostnamePort = config.get( gangliaServer );
-            final GMetric ganglia = new GMetric(
-                    hostnamePort.getHost(),
-                    hostnamePort.getPort(),
-                    GMetric.UDPAddressingMode.MULTICAST,
-                    1 );
-
-            gangliaReporter = GangliaReporter.forRegistry( registry )
-                                             .prefixedWith( prefix )
-                                             .convertRatesTo( TimeUnit.SECONDS )
-                                             .convertDurationsTo( TimeUnit.MILLISECONDS )
-                                             .filter( MetricFilter.ALL )
-                                             .build( ganglia );
-        }
+        // Setup Ganglia reporting
+        final GMetric ganglia = new GMetric( hostnamePort.getHost(), hostnamePort.getPort(), MULTICAST, 1 );
```

Comment:
Ganglia was added mostly as a checkbox ("Yes there's Ganglia"), as the metrics library had support for it, so there's no real design thought behind that. You could probably change that to simply be normal TCP instead (or whatever the options are), and it'll be fine. I'm personally using Graphite now in my app, so don't have experience with Ganglia. As I've said before, this is something that should be asked to those lovely people who do ops on Neo4j, to get feedback on what the real-life needs are. I don't think that survey has been done yet, properly.